src/api/status/public/index.html

```diff
@@ -355,66 +303,17 @@ <h4 class="mb-0" id="total-contributors-satellite">&nbsp;</h4>
                     /></a>
                   </div>
                   <div class="d-flex flex-column justify-content-center">
-                    <p class="mb-0">
-                      <a
-                        href=""
-                        title="User"
-                        target="_blank"
-                        id="latest-author-satellite"
-                        class="text-capitalize"
+                    <p class="mb-0 text-capitalize">
+                      <a href="" title="User" target="_blank" id="latest-author-satellite"
                         >&nbsp;</a
                       >
-                      <a
-                        href=""
-                        title="Commit"
-                        target="_blank"
-                        id="latest-author-commit-satellite"
-                        class="font-monospace btn-link text-xs"
-                      ></a>
```

Comment:
This is to show the latest commit, don't delete this.

Reply:
Oh, I only removed the li tags in the sidebar. Maybe because my version was not the latest one then...I will add it